content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -23,9 +23,12 @@ You should be familiar with [PKI certificates and requirements in Kubernetes](/d
 
 ## Renew certificates with the certificates API
 
-Kubeadm can renew certificates with the `kubeadm alpha certs renew` commands.
+The Kubernetes certificates reach normally their expiration date after one year. 
+
+Kubeadm can renew certificates with the `kubeadm alpha certs renew` commands; you should run these commands on Master only.
```

Comment:
@MalloZup Can you provide the reason why they should be done on the master?

Reply:
sadly our procedure for rotation is quite undocumented.
i can confirm that this has to be done on control plane nodes only.

with regards to your PR @MalloZup 

please replace:
`you should run these commands on Master only`
with:
`you should run these commands on control-plane nodes only`